src/pages/Admin/Charity/Templates/account/EndowmentStatus/useUpdateStatus.ts

```diff
@@ -25,17 +25,29 @@ export default function useUpdateStatus() {
 
   async function updateStatus(data: EndowmentUpdateValues) {
     if (!data.prevStatus) {
-      showModal(Popup, { message: "Endowment not found" });
-      return;
+      return showModal(Prompt, {
+        type: "error",
+        title: "Update Status",
+        headline: "Endowment not found",
+      });
     } else if (data.prevStatus === "closed") {
-      showModal(Popup, { message: "Endowment is closed and can't be changed" });
+      return showModal(Prompt, {
```

Comment:
Added a `return` statement here as it seemed like it was missing by accident, or was the original fine (no `return`)? 
@SovereignAndrey @ap-justin

Reply:
fine without return since no code after this if else block, but no big deal to add

Reply:
> no code after this if else block

What about the code on [line 54](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.8/src/pages/Admin/Charity/Templates/account/EndowmentStatus/useUpdateStatus.ts#L54), it's after the `if/else` block? :open_mouth: 
```ts
...
const [beneficiary, beneficiaryMeta] = (function (): [Beneficiary, string] {
...